src/features/PassSurvey/hooks/useSubmitAnswersMutation.ts

```diff
@@ -53,7 +67,20 @@ export const useSubmitAnswersMutations = ({ isPublic, onSubmitSuccess, onSubmitE
       }
     }
 
-    Mixpanel.track(event);
+    Mixpanel.track(assessmentCompletedEvent);
+
+    if (prolificParams) {
+      addFeatureToEvent(assessmentCompletedEvent, MixpanelFeature.Prolific);
+
+      const signupEvent: WithProlific<SignupSuccessfulEvent> = {
+        action: MixpanelEventType.SignupSuccessful,
+      };
```

Comment:
I think this `addFeatureToEvent` should apply to `signupEvent`, right? The feature was already added to `assessmentCompletedEvent` in line 42.

```suggestion
      const signupEvent: WithProlific<SignupSuccessfulEvent> = {
        action: MixpanelEventType.SignupSuccessful,
      };
      addFeatureToEvent(signupEvent, MixpanelFeature.Prolific);

```